portal/common/lib/logger.ts

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+/// Provides a simple logger interface function for
+/// logging messages on different runtimes.
+type LoggingPredicate = (...args: any) => void;
+
+/**
+ * Logger used as an abstraction for logging messages on different runtimes.
+ */
+ class Logger {
+    private infoPredicate: LoggingPredicate;
```

Comment:
Question: is there no level below info? i'd assume something like `debug` should exist?

Reply:
Created a new PR to avoid conflicts with the other branches https://github.com/MystenLabs/walrus-sites/pull/307